munch.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Rik",
+			"oem": "Poco",
+			"device": "POCO F4",
+			"filename": "crDroidAndroid-15.0-20241102-munch-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/munch/11.x/crDroidAndroid-15.0-20241102-munch-v11.0-BETA.zip/download",
+			"timestamp": 1730572087,
+			"md5": "9b02ad9868a69bc6083630fd20071caf",
+			"sha256": "27d479c4dbf84eb3ff041c42f82edf9f4094f3766eba1c08c38a5310b534a18a",
+			"size": 1439340224,
+			"version": "11.0",
+			"buildtype": "BETA",
+			"forum": "https://xdaforums.com/t/rom-android15-crdroid-official-poco-f4-redmi-k40s-munch-always-latest-build.4694105",
```

Comment:
update to new forum template
also lots of links are pointing to bad places, so correct that